mkosi/__init__.py

```diff
@@ -532,6 +532,12 @@ class CommandLineArguments:
 GPT_HEADER_SIZE = 1024 * 1024
 GPT_FOOTER_SIZE = 1024 * 1024
 
+# for alternate architectures we need to know which are compatible with host
+# and which are 'foreign' (e.g. aarch64 on x86_64 is foreign)
+ARCHITECTURES_COMPAT = {
```

Comment:
not a fan of needless abbreviations. Why not name this COMPATIBLE_ARCHITECTURES or so?